omero/sysadmins/server-upgrade.txt

```diff
@@ -280,6 +280,31 @@ intermediate major release.
    connected to the database using **db_user** before running the script. See
    :forum:`this forum thread <viewtopic.php?f=5&t=7778>` for more information.
 
+Delete certain annotations (optional)
+"""""""""""""""""""""""""""""""""""""
+
+For various reasons production databases may accumulate non-sharable
+annotations that are orphaned. These are :doc:`structured annotations
+</developers/Model/StructuredAnnotations>` that are 'basic' (`Boolean`,
+`TimeStamp`, `Term`), 'numeric' (`Double`, `Long`), or `Comment`, and
+that are *not* annotating an object. An illustrative example is that
+deleting a rating in OMERO.insight 5.2 may have left behind the
+corresponding `Long` annotation that captured the rating's number of
+stars. The script does *not* delete annotations that have a
+custom/non-OME namespace (ns) set.
```

Comment:
I think it would be worth to mention in this rationale that we are talking aoubt ``annotations which cannot be accessed using OMERO.clients anymore, and thus might be an artifact of the workflow rather than useful data.